lib/client/client.rb

```diff
@@ -4,40 +4,37 @@
 require "json"
 
 class ApiClient
-   ENDPOINT = "https://api.replicated.com/vendor/v1"
+   VENDOR_ENDPOINT = "https://api.replicated.com"
```

Comment:
vendor/v1?

Reply:
nvm, we are only using the host and port. May be better to just change this to `VENDOR_HOST` and add a `VENDOR_PORT` const